stock_reception_screen_measuring_device/models/stock_reception_screen.py

```diff
@@ -30,13 +36,17 @@ class StockReceptionScreen(models.Model):
         store=True,
     )
 
-    @api.depends("product_packaging_id", "product_packaging_id.measuring_device_id")
+    @api.depends(
+        "current_move_product_id.packaging_ids.measuring_device_id",
```

Comment:
We could trigger a lot of computation with such depends knowing that `scan_requested` field is `store=True`, especially each time we call `_measuring_device_assign`/`_measuring_device_release` methods.

IMO this `scan_requested` field could be set `store=False` as it's only used to display a button on the view. So this field will be computed only when needed, avoiding recomputation of its value on all `stock.reception.screen` records.

Reply:
@mmequignon this sounds like a good point